test/helpers/Factory.ts

```diff
@@ -468,7 +468,13 @@ const setupPool = async (
         await createPool(reserveToken, network, networkSettings, poolCollection);
 
         await networkSettings.setFundingLimit(reserveToken.address, MAX_UINT256);
-        await poolCollection.requestFundingT(formatBytes32String(''), reserveToken.address, spec.requestedLiquidity);
+        if (spec.requestedLiquidity) {
```

Comment:
unrelated to this PR: should be `requestedFunding`

Reply:
👍